smart_card_connector_app/src/window-main.js

```diff
@@ -29,6 +29,7 @@ goog.require('GoogleSmartCard.ConnectorApp.Window.LogsExporting');
 goog.require('GoogleSmartCard.I18n');
 goog.require('GoogleSmartCard.Logging');
 goog.require('GoogleSmartCard.InPopupMainScript');
```

Comment:
BTW, looks like when #388 renamed the "Client"/"Server" modules, these lexicographic orders were broken. Let's sort the imports now.